pkgs/top-level/all-packages.nix

```diff
@@ -727,6 +727,8 @@ with pkgs;
 
   mix2nix = callPackage ../development/tools/mix2nix { };
 
+  migra = callPackage ../tools/misc/migra { };
```

Comment:
Please move the package to `pkgs/by-name/mi/migra/package.nix`, so that this line can be removed.